test/router.spec.js

```diff
@@ -66,172 +66,166 @@ describe('the router', () => {
   });
 
   describe('generate', () => {
-    it('should generate route URIs', () => {
+    it('should generate route URIs', (done) => {
```

Comment:
There are two new tests. All other changes to tests are just moving assertions into `Promise` continuations to handle the new async `Router.configure` implementation.